src/main/java/Duke.java

```diff
@@ -1,10 +1,120 @@
+                }catch (Exception e){
+                    System.out.println("\tOOPS!!! Please input the index.\n" + LINE + "\n");
+                }
+            }
+            else if (splittedCmd[0].equals("unmark")) {
+                try {
+                    System.out.println(LINE);
+                    int idx = Integer.parseInt(splittedCmd[1]) - 1;
+                    Task task = list.get(idx);
+                    System.out.println("\tNice! I've unmarked this task as not done yet:");
+                    task.unmark();
+                    System.out.println("\t" + task.toString());
+                    System.out.println(LINE + "\n");
+                    list.set(idx, task);
+                } catch (Exception e) {
+                    System.out.println("\tOOPS!!! Please input the index.\n" + LINE + "\n");
+                }
+            }
+            else if (splittedCmd[0].equals("delete")){
+                try {
+                    System.out.println(LINE);
+                    int idx = Integer.parseInt(splittedCmd[1]) - 1;
+                    Task task = list.get(idx);
+                    list.remove(idx);
+                    System.out.println("\tNoted. I've removed this task:");
+                    System.out.println("\t" + task.toString());
+                    System.out.println("\t" + "Now you have " + list.size() + " tasks in the list");
+                    System.out.println(LINE + "\n");
+                }catch (Exception e){
+                    System.out.println("\tOOPS!!! Please input the index.\n" + LINE + "\n");
+                }
+            }
+            else if (splittedCmd[0].equals("todo")){
+                try {
+                    System.out.println(LINE);
+                    ToDos todos = new ToDos(splittedCmd[1]);
+                    list.add(todos);
+                    System.out.println("\t" + "Got it. I've added this task:");
+                    System.out.println("\t" + todos.toString());
+                    System.out.println("\t" + "Now you have " + list.size() + " tasks in the list");
+                    System.out.println(LINE + "\n");
+                }catch(Exception e){
+                    System.out.println("\tOOPS!!! The description of a todo cannot be empty.\n" + LINE + "\n");
+                }
+            }
+            else if (splittedCmd[0].equals("deadline")){
+                try {
+                    System.out.println(LINE);
+                    String[] request = splittedCmd[1].split("/", 2);
+                    String task = request[0];
+                    String date = request[1];
+                    Deadlines deadlines = new Deadlines(task, date);
+                    list.add(deadlines);
+                    System.out.println("\t" + deadlines.toString());
+                    System.out.println("\t" + "Now you have " + list.size() + " tasks in the list");
+                    System.out.println(LINE + "\n");
+                }catch(Exception e) {
```

Comment:
Perhaps you could add spacing between the curly bracket and the catch word, to better follow coding standards.